.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
do we want the .vscode files checked in?

Reply:
yes, this will help prettier run correctly and clean up code.